posthog/hogql_queries/web_analytics/web_overview.py

```diff
@@ -190,60 +211,68 @@ def previous_period_aggregate(
                 params=params,
             )
 
+        def add_metric_pair(
+            metrics_list: list[ast.Expr],
+            function_name: str,
+            column_name: str,
+            current_alias: str,
+            previous_alias: Optional[str] = None,
+            params: Optional[list[ast.Expr]] = None,
+        ) -> list[ast.Expr]:
+            # This could also be done using tuples like the stats_table but I will keep the protocol as close as possible: https://github.com/PostHog/posthog/blob/26588f3689aa505fbf857afcae4e8bd18cf75606/posthog/hogql_queries/web_analytics/stats_table.py#L390-L399
+            previous_alias = previous_alias or f"previous_{current_alias}"
+            metrics_list.append(current_period_aggregate(function_name, column_name, current_alias, params))
+            metrics_list.append(previous_period_aggregate(function_name, column_name, previous_alias, params))
+            return metrics_list
```

Comment:
This slightly changes the column names, standardizing them but they are not used on the frontend, we rely on the `{value, previous}` the `to_data` method transforms the results to.